components/generic/PaginationNav.vue

```diff
@@ -64,3 +64,17 @@
     }
   };
 </script>
+
+
+<style lang="scss">
+  @import "./assets/scss/variables.scss";
+
+  .page-link {
+    color: black;
+  }
```

Comment:
Use the $black here as well. 
Or is it then complaining about the contrast?

Reply:
works